users app, fixtures and test cases

scripterkaran opened the pull request.

scripterkaran committed:
users app, fixtures and test cases

scripterkaran committed:
updated readme , added examples in curl

scripterkaran committed:
alter phone_number field to have char field

utkbansal reviewed (commented):
@scripterkaran Can you please add validation for phone numbers? Currently, we can `foo=bar` as the phone number. If needed, you can assume the phone number to be US format.

scripterkaran committed:
added back validation for phone number field

scripterkaran commented:
@utkbansal I had added it before, but had to remove it because of inconsistent validations on frontend  for phone number. (By trying different libraries)

But now I have added Phonenumber [package](https://github.com/daviddrysdale/python-phonenumbers), which handles all the conditions required. 

More work was required at frontend to handle the HTML use case where `+` in HTML `input[type=number]` was not a valid input range.
